From a236b4b8f343725bc84d31594163147adf2af3db Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 7 Feb 2016 10:19:44 +0200
Subject: [PATCH 0818/1240] config: armada7040: enabled interfaces in
 armada7040 board config

enabled the IOB, ICU, AHCI, DW PCI and MDIO interfaces

Change-Id: I8c463461c6f76dde8653b3b92050d1bc0f943716
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27260
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 configs/mvebu_armada70x0_defconfig | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/configs/mvebu_armada70x0_defconfig b/configs/mvebu_armada70x0_defconfig
index 9a0fafa..abac2ab 100644
--- a/configs/mvebu_armada70x0_defconfig
+++ b/configs/mvebu_armada70x0_defconfig
@@ -3,10 +3,12 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_8K=y
 CONFIG_MVEBU_CCU=y
+CONFIG_MVEBU_IOB=y
 CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
++S:# CONFIG_MULTI_DT_FILE is not set
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
@@ -14,9 +16,11 @@ CONFIG_CMD_SAVEENV=y
 CONFIG_CMD_MEMORY=y
 CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_LOADB=y
+CONFIG_CMD_PCI=y
 CONFIG_CMD_ECHO=y
 CONFIG_CMD_NET=y
 CONFIG_CMD_PING=y
+CONFIG_CMD_MII=y
 CONFIG_CMD_TIME=y
 CONFIG_CMD_MISC=y
 CONFIG_CMD_TIMER=y
@@ -27,10 +31,15 @@ CONFIG_CMD_MVEBU_TSEN=y
 +S:CONFIG_OF_CONTROL=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="armada-7040-rz-db_router"
-CONFIG_MVEBU_PCI=y
+CONFIG_MVEBU_DW_PCIE=y
+CONFIG_SCSI_AHCI_PLAT=y
+CONFIG_MVEBU_ICU=y
+CONFIG_MVEBU_MDIO=y
 CONFIG_ENC28J60=y
+CONFIG_PHYLIB=y
+CONFIG_PHY_MARVELL=y
 +S:CONFIG_SYS_NS16550=y
-+S:CONFIG_MVEBU_I2C=y
+CONFIG_MVEBU_I2C=y
 CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_PINCTL=y
 CONFIG_USB=y
-- 
1.9.1

